macros/upload_results.sql

```diff
@@ -109,14 +108,21 @@
 
         {% do log("Uploading models", true) %}
         {% set models = dbt_artifacts.get_relation('models') %}
-        {% set content_models = dbt_artifacts.upload_models(graph) %}
-        {{ dbt_artifacts.insert_into_metadata_table(
-            database_name=models.database,
-            schema_name=models.schema,
-            table_name=models.identifier,
-            content=content_models
-            )
-        }}
+        {% set models_set = [] %}
+        {% for node in graph.nodes.values() | selectattr("resource_type", "equalto", "model") %}
+            {% do models_set.append(node) %}
+        {% endfor %}
+        {% set upload_limit = 50 if target.type == 'bigquery' else 5000 %}
```

Comment:
Might be worth making it lower for other warehouses too?
```suggestion
        {% set upload_limit = 50 if target.type == 'bigquery' else 100 %}
```